docs/openapi.yaml

```diff
@@ -203,7 +227,10 @@ paths:
             application/json:
               examples:
                 error_104_example:
-                  $ref: "#/components/examples/error_104_example"
+                  value:
+                    error_code: 104
+                    message: Invalid syntax
+                    details: [The provided user ID is invalid]
```

Comment:
invalid is not found or invalid format (ex: id > 0 ) ??

Reply:
đã update